src/controllers/hero-list.js

```diff
@@ -0,0 +1,12 @@
+const heroListService = require('../services/herolist');

```

Comment:
Unfortunate naming. The 'list' suggest the used data structure, if the structure changes for example to a map or a database, then this name would be inaccurate.
Change it to heroService, and follow up the changes in the project.